src/layouts/Footer/components/Footer-Container/FooterContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+import NavLink from './components/NavLink/NavLink';
+import ScrollUpButton from './components/ScrollUpButton/ScrollUpButton';
+
+
+
+const FooterContainer: FC = () => {
+    
+    return (
+        <div className='bg-[#03111B] text-white h-96 mx-auto flex flex-col justify-center max-w-7xl'>
+        <div className='text-base space-x-9 pt-28 mx-auto'>
+              <NavLink Ref="/features" LinkName="Features"/>
+              <NavLink Ref="/pricing" LinkName="Pricing"/>
+              <NavLink Ref="/contact" LinkName="Contact"/>
+        </div>
+    
+        <div className='text-xs pt-16 mx-auto'>
+       © Report Builder, 2023 - All Rights Reserved 
+        </div>
+    
+        <div className='justify-center items-center z-10 pt-14 mx-auto'>
```

Comment:
```suggestion
        <div className='z-10 pt-14 mx-auto'>
```

justify-center items-center 只有在 flex的情况下才会起作用

Reply:
Resolved